docs/api-types/index.md

```diff
@@ -141,6 +141,7 @@ KeylessRef contains location of the validating certificate and the identities ag
 | url | URL defines a url to the keyless instance. | apis.URL | false |
 | identities | Identities sets a list of identities. | [][Identity](#identity) | false |
 | ca-cert | CACert sets a reference to CA certificate | [KeyRef](#keyref) | false |
+| trustRootRef | Use the Certificate Chain from the referred TrustRoot.CertificateAuthorities and TrustRoot.CTLog | string | false |
```

Comment:
I thought you wrote `TrustRoot.TLog` in the description. You use TrustRoot.CTLog here.

Reply:
I see you refer to the keyless, so you use CTLog although I will assume you refer to the rekor instance instead and so use TLog.

Reply:
Well, this one is actually the CTLog, since it's the CA for Fulcio, and the CTLog for the Trillian that Fulcio writes to.